src/Cleave.react.js

```diff
@@ -23,7 +23,7 @@ var cleaveReactClass = CreateReactClass({
         Util.setSelection(owner.element, owner.state.cursorPosition, pps.document);
     },
 
-    componentWillReceiveProps: function (nextProps) {
+    UNSAFE_componentWillReceiveProps: function (nextProps) {
```

Comment:
#

Reply:
Doing interaction i've experience a props make you mislead and then when commit an error on the recent pages your information you created will disaffear in that case your personal information including credit card numbers names remain in the files hackers easily download your document and using for fraud..